### ISSUE TITLE
Wrong map extent for maps using layers with different CRS

### ISSUE THREAD

**Issue by NyakudyaA** [kartoza/geonode#489]
# Problem
After the fix for the CRS I successfully managed to upload layers in UTM but when I try to use them in conjunction with EPSG:4326 to create a map the extent is not correct

Current map
![current](https://user-images.githubusercontent.com/2510900/45350396-00628f00-b5b4-11e8-8586-ae302ef85b79.png)

After adding layer with UTM
![utm33](https://user-images.githubusercontent.com/2510900/45350436-11ab9b80-b5b4-11e8-965c-4b199a783b42.png)

# Solution
Please fix

**Comment by lucernae** [comment]
@boney-bun can you comment on this?

**Comment by NyakudyaA** [comment]
This is still an issue on testing.

* Layer in 3857
![3857](https://user-images.githubusercontent.com/2510900/46721070-779e4800-cc72-11e8-9aec-75f162895903.png)

* Layer details page
![layer-detail](https://user-images.githubusercontent.com/2510900/46721107-8ab11800-cc72-11e8-889d-889142155ca4.png)

**Comment by lucernae** [comment]
CMIIW, so the one that is still broken is on Map page right?

**Comment by NyakudyaA** [comment]
Yes @lucernae The layer loads fine but just the map page is buggy I suspect when you fixed the CRS thing it only applied to the uploads
